### PR TITLE
fix(editor): Fix trigger node type identification on add to canvas

### PR DESCRIPTION
**Pull request by ivov**
https://n8nio.slack.com/archives/C035KBDA917/p1672171809503329?thread_ts=1672154878.519009&cid=C035KBDA917

### CODE REVIEW

File: packages/editor-ui/src/stores/nodeCreator.ts [hunk 1]
```diff
@@ -339,8 +339,7 @@ export const useNodeCreatorStore = defineStore(STORES.NODE_CREATOR, {
 				if (!nodeType) return [];
 
 				const { workflowTriggerNodes } = useWorkflowsStore();
-				const isTrigger =
-					nodeType.toLocaleLowerCase().includes('trigger') || nodeType === WEBHOOK_NODE_TYPE;
+				const isTrigger = useNodeTypesStore().isTriggerNode(nodeType);
```
Comment (cstuncsik) [inline packages/editor-ui/src/stores/nodeCreator.ts:342]: I'd save `useNodeTypesStore()` call result to a variable but for that you should refactor the creation of this store to use a callback function that returns the object (which is now passed directly as the second argument of `defineStore`)

Comment (cstuncsik) [reply packages/editor-ui/src/stores/nodeCreator.ts:342]: Never mind, maybe switching to the `setup` store creation pattern is not what you want to do just right now :)

**Review by cstuncsik — APPROVED** [review]

File: packages/editor-ui/src/stores/nodeCreator.ts (hunk 1 above)
Comment (ivov) [reply packages/editor-ui/src/stores/nodeCreator.ts:342]: Thanks! Exactly

**Comment by janober** [comment]
Got released with `n8n@0.209.4`

**Pull request merged**